lib/CL/pocl_llvm_build.cc

```diff
@@ -546,9 +546,6 @@ int pocl_llvm_build_program(cl_program program,
 
 #ifdef LLVM_OLDER_THAN_3_9
   (*mod)->setPICLevel(PICLevel::Large);
```

Comment:
any reason why it's enabled for pre-3.9 LLVM but disabled for newer ? if not, we should probably keep it consistent.